otelopscol/build.ps1

```diff
@@ -54,9 +54,8 @@ powershell.exe -Command $ocbGenerateCommand
 # Build the collector.
 $ldFlags="-s -w"
 if ($jmxHash -ne "") {
-    $ldFlags+=" -X github.com/open-telemetry/opentelemetry-collector-contrib/receiver/jmxreceiver.MetricsGathererHash=$jmxHash"
+    $ldFlags+=" github.com/open-telemetry/opentelemetry-collector-contrib/receiver/jmxreceiver.MetricsGathererHash=$jmxHash"
```

Comment:
This needs to reverted, back to master

Reply:
@ridwanmsharif found a bug, which was fixed in https://github.com/GoogleCloudPlatform/opentelemetry-operations-collector/pull/278.